src/main/resources/assets/js/models/Router.js

```diff
@@ -0,0 +1,33 @@
+define([
+  "Backbone"
+], function(Backbone) {
+  "use strict";
+
+  var noop = function () {};
+
+  return Backbone.Router.extend({
+
+    routes: {
+      "": "apps",
+      "apps": "apps",
+      "deployments": "deployments",
+      "app/:appid(/:view)": "app",
```

Comment:
Can we make apps and apps/:appid and apps/:appid/:view one function?
Like this maybe `apps(/:appid/:view)` and the function will be something like this:
```
routeApps: function(appid, view) {
  if (appid != null) {
    ...
  }
  if (view != null) {
    ...
  }
  ...
}
```